src/components/RGBColorPicker/RGBColorPicker.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import {SingleColorPicker} from '../SingleColorPicker/SingleColorPicker';
+
+class RGBColorPicker extends Component {
```

Comment:
everything is working properly you just missing the part where you show the background color with the rgb color picked.